src/tests/ftest/security/pool_security_acl.py

```diff
@@ -0,0 +1,400 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2018-2019 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+'''
+from __future__ import print_function
+
+import os
+from apricot import TestWithServers
+from daos_utils import DaosCommand
+from dmg_utils import DmgCommand
+import dmg_utils
+import random
+import pwd
+import grp
+import re
+from general_utils import pcmd
+
+
+PERMISSIONS = ["", "r", "w", "rw"]
+
+def acl_entry(usergroup, name, permission):
+    '''
+    Deascription:
+        Create a daos acl entry on the specified user or groups
+        with specified or random permission.
+    Args:
+        usergroup: user or group.
+        name: user or group name to be created.
+        permission: permission to be created.
+    Return:
+        entry: daos acl entry.
+    '''
+    if PERMISSIONS == "random":
+        permission = PERMISSIONS[random.randint(0, 3)]
```

Comment:
Just a suggestion, maybe should use len(PERMISSIONS) as the end of the range. More permissions will be added in the future.

Reply:
Thanks for the review and comment..
Please see the test script update for the fixes...